lib/core/datatable/components/datatable/datatable.component.html

```diff
@@ -110,14 +110,16 @@
                                     <ng-template #no_selected_row>
                                         <img class="adf-datatable-center-img-ie"
                                             [attr.aria-label]="data.getValue(row, col) | fileType"
-                                            alt="{{ iconAltTextKey(data.getValue(row, col)) | translate }}"
+                                            alt="{{ 'ADF-DATATABLE.ACCESSIBILITY.ICON_ALT_TEXT' | translate:{ type: data.getValue(row, col) | fileType } }}"
```

Comment:
are all the translations available for the file types?

Reply:
should be covered when [this](https://github.com/Alfresco/alfresco-ng2-components/pull/4998/files/35a116419ac078986712eb0c61bb6872026c4bda#diff-e43039c591eae18181382500e64ba7e8R112) was done I guess

Reply:
can then that work be included into this PR? it does not make much sense splitting that into multiple PRs

Reply:
agree

Reply:
I agree with @DenysVuika